includes/Story_Post_Type.php

```diff
@@ -113,6 +113,15 @@ public static function get_requirements(): array {
 		return [ 'settings' ];
 	}
 
+	/**
+	 * Get settings.
+	 *
+	 * @return Settings Settings instance.
+	 */
+	public function get_settings(): Settings {
+		return $this->settings;
```

Comment:
@spacedmonkey @swissspidy I've added a getter for setting for being able to use it for the dashboard. 
LMK if you prefer some other way, or feel free to amend to the PR directly.

Reply:
We haven't done this in other places. Getters like this can get messy. Let's just pass settings into constructor like other classes.

Reply:
Good point, I can see now that you're doing that everywhere else. Will change, thanks!